docs/README.rst

```diff
@@ -0,0 +1 @@
+../README.md
```

Comment:
Is this a placeholder? If this is meant to transclude the markdown README you might need `mdinclude` here, and from https://github.com/readthedocs/recommonmark it looks like you'll need this in `docs/conf.py`:

```
source_parsers = {
    '.md': CommonMarkParser,
}
```

Reply:
Good point. Better yet, the markdown can be converted to reST to conform with other API scheme.